src/runner.py

```diff
@@ -187,7 +197,28 @@ def run(self, args):
 class cmd_loop(Command):
     help = "Listen to pub/sub events and run in a loop"
     args = [Args.db_config, Args.lab_config, Args.output]
-    opt_args = [Args.verbose, Args.plan]
+    opt_args = [
+        Args.verbose,
+        Args.plan,
+        Args.target,
+        {
+            'name': '--ssh-host',
+            'help': "Storage SSH host",
+        },
+        {
+            'name': '--ssh-key',
+            'help': "Path to the ssh key for uploading to storage",
+        },
+        {
+            'name': '--ssh-port',
+            'help': "Storage SSH port number",
+            'type': int,
+        },
+        {
+            'name': '--ssh-user',
+            'help': "Storage SSH user name",
+        },
```

Comment:
Since these are the common command line arguments for `cmd_loop` and `cmd_run`, maybe we can store them in a separate list and use it to extend `opt_args` to drop the number of duplicate lines in the code.

Reply:
Yes, that's fine.